mlir/lib/Dialect/SparseTensor/Transforms/Utils/SparseTensorLevel.h

```diff
@@ -119,8 +132,8 @@ class SparseIterator {
   virtual Value upperBound(OpBuilder &b, Location l) const = 0;
 
   // Serializes and deserializes the current status to/from a set of values. The
-  // ValueRange should contain values that specifies the current postion and
-  // loop bound.
+  // ValueRange should contain values that are suffcient to recover the current
```

Comment:
typo: sufficient

also, sufficient for what?

Reply:
to recover the current iterator position as well as loop bound?

Reply:
Ah, I really meant, explain sufficient for what in the doc itself ;-)